Misc/NEWS.d/next/Library/2022-01-26-18-06-08.bpo-46534.vhzUM4.rst

```diff
@@ -0,0 +1 @@
+Implement PEP 673 :class:`typing.Self`.
```

Comment:
Can add "Patch by James Hilton-Balfe" if you like.

Reply:
Please do, that' very conventional.